### ISSUE TITLE
Screen reader is not announcing that some links in the marketplace view are "clickable"

### ISSUE THREAD

**Issue by andschwa** [microsoft/vscode#135052]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Version: 1.61.0 (Universal)
Commit: ee8c7def80afc00dd6e593ef12f37756d8f504ea
Date: 2021-10-07T18:11:58.853Z
Electron: 13.5.1
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin arm64 20.6.0
- OS Version: macOS Big Sur version 11.6 (20G165) (and other OSes)

Steps to Reproduce:

1. Start a screen reader
2. Open Code
3. Open marketplace within in Code
4. Open extension view in marketplace (in Code)
5. Press "tab" to tab through the links and other items in the extension view's header

Each clickable link should be announced _as clickable_ (screen reader's will render this differently, macOS's voice over says "you are on a button" (or link) and then how to click it). However, multiple clickable items in the extension view are not properly annotated in the source as clickable. Notably (and what was reported to us as maintainers of the PowerShell extension) the "install count" is not annotated, so screen readers are not announcing it can be clicked. However, just while producing a fresh screen grab for you, I noticed that the "publisher" is also not annotated.

I am _not_ a VS Code developer, and know very little about this, but my quick investigation led me to here:

https://github.com/microsoft/vscode/blob/2e0c53def86aad0f6cd2d86e168a1ad87d9ee0da/src/vs/workbench/contrib/extensions/browser/extensionEditor.ts#L225-L228

and it looks like there are a number of clickable items (e.g. `installCount`) that need to be fixed and properly annotated as clickable.

See this screen grab for more:

https://user-images.githubusercontent.com/2226434/137230613-1b9aeda0-ca49-4353-9db7-4496567ddf5e.mov

**Comment by isidorn** [comment]
`Install Count` is not clickable thus it is not annotated. 
`Publisher` is clickable and thus it has the role `button` set.
All the other items are I think annotated properly - if there are some which you think are not properly annotated please let me know.

Closing as dup of https://github.com/microsoft/vscode/issues/132645
